### ISSUE TITLE
[Feature Req] Worldometers Weekly Trends data source

### ISSUE THREAD

**Issue by asallay** [disease-sh/API#995]
There is a new table for the weekly trends data in the Worldometer Coronavirus updates page: 

https://www.worldometers.info/coronavirus/weekly-trends/#countries

It would be nice to get this data in the API too, as a separate endpoint, or merged into the regular worldometers based one. 
Especially since the EU's recommendation for the restriction of free movement in response to the COVID-19 pandemic is partly based a moving average of 14 days' new cases, and it could be easily calculated dynamically from this data set on a daily basis.

**Comment by ebwinters** [comment]
I agree this is useful information. The work that is needed here involves:
1. Writing a scraper that reuses some code to parse this HTML table and throw the data into Redis
2. Writing tests
3. Writing API docs

We won't be calculating anything further than the data that is there, since we are just a source. I don't have time to work on this right now, but let me post in Discord and see if anyone is interested (assuming you are not)

**Comment by puf17640** [comment]
Ayy this is really interesting!!! I'm not gonna promise anything but if I ever get time to work on a new feature again, this will be it!
